.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-    - 5.5
```

Comment:
We could check against PHP 5.5 **and** PHP 5.6. But why should we check only against PHP 5.5 on Symfony 2.3?